src/core/worker.js

```diff
@@ -198,218 +198,123 @@ IPDFStreamRangeReader.prototype = {
 
 /** @implements {IPDFStream} */
 var PDFWorkerStream = (function PDFWorkerStreamClosure() {
-  function PDFWorkerStream(params, msgHandler) {
-    this._queuedChunks = [];
-    var initialData = params.initialData;
-    if (initialData && initialData.length > 0) {
-      this._queuedChunks.push(initialData);
-    }
+  function PDFWorkerStream(data, msgHandler) {
```

Comment:
Actually, we don't need to pass `source` or `data` to `PDFWorkerStream` here, this can changed to accept `msgHandler` as it's single parameter, something like: 
 `function PDFWorkerStream(msgHandler) {...`.